app/javascript/components/Editor.js

```diff
@@ -14,13 +14,21 @@ class Editor extends React.Component {
 
     const getView = () => this.view
 
+    if (this.props.field === 'title') {
+      this.plugins = props.options.titleSetupPlugins(getView)
```

Comment:
maybe dont put it in `this.plugins` but just `let plugins`

Reply:
it may be better to move all of this into `setupPlugins()` and pass in the field

Reply:
I refactored `Editor.js` by setting `titlePlugins` and `titleSetupPlugins` object property names to `setupPlugins:` in `index.js`. This removes the need to set plugins for title editor and body editor separately.

I moved the `titleSetupPlugins` function into the `plugins.js` file.

Do you think I should find a way to combine the `titleSetupPlugins` and `setupPlugins` into a single function?

Reply:
yeah thats what i was thinking - just turn setupPlugins into a function that returns the hardcoded list of plugins for the different title and body editors.

Reply:
e.g. https://github.com/jellypbc/poster/pull/315/files#r502899998

Reply:
> turn setupPlugins into a function that returns the hardcoded list of plugins for the different title and body editors

I don't know how to achieve this. Can you say more about how you would implement this?

`setupPlugins` takes in `getView` and returns a function because the `citationUI` and `commentUI` need access to the current view. If we want to pass in `body` or `title` in `setupPlugins(body)` or `setupPlugins(title)` what do we do about `getView`?

I pushed up a commit to simplify, but I don't think this is what you intended.